build/common/installer/scripts/tomlparser.rb

```diff
@@ -116,6 +138,15 @@ def populateSettingValuesFromConfigMap(parsedConfig)
             end
           end
         end
+
+        if @collectStderrLogs && !stderrSystemPods.nil? && stderrSystemPods.kind_of?(Array)
+          # Checking only for the first element to be string because toml enforces the arrays to contain elements of same type
+          if stderrSystemPods.length > 0 && stderrSystemPods[0].kind_of?(String)
+            @stderrIncludeSystemPods = stderrSystemPods.join(",")
+            puts "config::Using config map setting for stderr log collection to include system pods"
```

Comment:
same here .. If customer uses the namespaces other than system namespaces list, we should log warn message and ignore the entry.